app/javascript/app/utils/map.js

```diff
@@ -0,0 +1,24 @@
+export const getDefaultStyles = () => ({
```

Comment:
My problem with this file inside utils is that when piece of functionality is not used in more than one component, by moving it to the top level you are breaking the modularisation of that component without any benefit. We have a couple of options here, either you have a utils file inside the module's folder, or you can also declare that function in the file where it's used.